lib/services/item_manager.ts

```diff
@@ -545,20 +545,24 @@ export class ItemManager extends PureService {
    * Duplicates an item and maps it, thus propagating the item to observers.
    * @param isConflict - Whether to mark the duplicate as a conflict of the original.
    */
-  public async duplicateItem(uuid: UuidString, isConflict = false) {
```

Comment:
So the reason we duplicate by uuid is that Item objects are immutable, and when a caller wants to duplicate an item, they typically mean "duplicate the latest values of this item", but the caller may be passing in a stale item reference. By looking up the item instead we ensure we are duplicating its latest value.